src/03_modules.py

```diff
@@ -9,23 +9,29 @@
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
+print("Command Line Args: ", sys.argv)
```

Comment:
If `sys.argv` returns a list, think about how you could print each argument on its own line. You can add arguments to the end of your terminal command (separated by spaces) when you run the file i.e. `python modules.py foo bar -a --whatever` and they'll show up in the `sys.argv` list.